env.go

```diff
@@ -593,8 +593,8 @@ func parseFieldParams(field reflect.StructField, opts Options) (FieldParams, err
 	return result, nil
 }
 
-func get(fieldParams FieldParams, opts Options) (val string, err error) {
-	var exists, isDefault bool
+func get(fieldParams FieldParams, opts Options) (val string, isDefault bool, err error) {
```

Comment:
I would return a struct then


```suggestion
type whatever struct {
    val             string
    isDefault bool
}

func get(fieldParams FieldParams, opts Options) (whatever, err error) {
```

Reply:
Done